ts/llm_launcher.py

```diff
@@ -26,74 +56,109 @@ def get_model_config(args):
         "responseTimeout": 1200,
         "deviceType": "gpu",
         "asyncCommunication": True,
```

Comment:
The vllm handler uses the async vllm engine to process multiple requests at a time. Does trt-lmm have something equivalent? If thats the case we should await the generate statement (or an async wrapper) in `def inference` to allow the other coroutine to be executed while generate is running. Otherwise we're only working on a single request at a time which defeats the purpose of the engine.

Reply:
Thanks. Done. This was supported with another API. Didn't have great documentation but got it working